release/models/mpls/openconfig-mpls-static.yang

```diff
@@ -24,7 +24,15 @@ submodule openconfig-mpls-static {
     "Defines static LSP configuration";
 
 
-  oc-ext:openconfig-version "3.6.0";
+  oc-ext:openconfig-version "3.7.0";
+
+  revision "2024-11-12" {
+    description
+      "Added support for mpls route post decap to a specific 
```

Comment:
Can you please clarify this? What is `post decap` in this case?

In terms of mpls static LSPs, you have three different scenarios:
1) ingress 
2) transit
3) egress

Does `decap` implies the egress (where the top label is removed by a PE node, and the LSP is terminated)?

Reply:
We should remove `post decap`. 
This is for egress LSPs. 
The intent is to do a lookup for the nexthop (to determine egress interface and rewrite) in a vrf that is different from the ingress interface's VRF.
For example: in some vpc use case ingress interface is in the default VRF as it's facing the providers network and the egress interface is in a customer specific VRF.

Reply:
May I kindly request 

> Ok. In that case, in addition to updating the description, I think this leaf should be moved from `static-lsp-common-config` to `static-lsp-egress-config`

I had updated description and can move leaf to egress config, may I kindly request if Nokia platform can support this? I could not find any reference from Nokia public documentation.

As a reference ARISTA support's this using following CLI command:
```
mpls static top-label vrf pop payload-type ipv4/6
```